### ISSUE TITLE
es6 template string throws a comma-spacing rule false positive

### ISSUE THREAD

**Issue by sidred10** [eslint/eslint#2126]
Given the code
````
line = `${base},${rank.d},${rank.r}`;
````

eslint throws an error 
````
Error - A space is required after ','. (comma-spacing)
````
 The comma-spacing rule should not be appllied within template strings

**Comment by nzakas** [comment]
Thanks, we will look into it.

**Comment by nzakas** [comment]
I took a look and was unable to reproduce with the latest version. What version are you using?

**Comment by sidred10** [comment]
I was using 0.16.1. Just upgraded to 0.17.1 and the error no longer shows up 

Thank You.

**Comment by sidred10** [comment]
just checked on 0.17.1. The error still shows up.

**Comment by sidred10** [comment]
Finally figured out the problem. I has parser set to "babel-eslint". Switching to the default parser, no longer throws the error.
